src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/amd64/AMD64HotSpotVMConfig.java

```diff
@@ -88,4 +91,6 @@ class AMD64HotSpotVMConfig extends HotSpotVMConfigAccess {
     final long amd64OSPKE = getConstant("VM_Version::CPU_OSPKE", Long.class);
     final long amd64CET_IBT = getConstant("VM_Version::CPU_CET_IBT", Long.class);
     final long amd64CET_SS = getConstant("VM_Version::CPU_CET_SS", Long.class);
+    final long avx10_1 = getConstant("VM_Version::CPU_AVX10_1", Long.class);
```

Comment:
Leave them as is. @mur47x111 plans to remove them [1].

[1] https://github.com/openjdk/jdk/pull/24329#issuecomment-2838223030